yarn.lock

```diff
@@ -0,0 +1,1170 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't know enough about yarn, but we currently do not intend to use it. Since we won't update `yarn.lock`, I think this file should not be in the repository?

Since you need it during the tests, you may either monkey-patch `existsSync` or write an actual lock file before the test (and remove it afterwards).

Then you can remove the `forceNpm: true` flag again. What do you think?